packages/toolkit/src/shared/awsClientBuilder.ts

```diff
@@ -11,6 +11,12 @@ import { DevSettings } from './settings'
 import { getUserAgent } from './telemetry/util'
 import { telemetry } from './telemetry/telemetry'
 
+/** Suppresses a very noisy warning printed by AWS SDK v2, which clutters local debugging output, CI logs, etc. */
+export function disableAwsSdkWarning() {
+    // See https://github.com/aws/aws-sdk-js/issues/4354
+    require('aws-sdk/lib/maintenance_mode_message').suppress = true
```

Comment:
Is it worth wrapping this in a try/catch in case the SDK does something that explodes?

Reply:
I thought about that but I think we want it to explode, when we remove v2 we don't need this anymore.